marimbabot_vision/launch/marimbabot.launch

```diff
@@ -1,10 +1,10 @@
 <launch>
 <node name="cv_camera_node" pkg="cv_camera" type="cv_camera_node" respawn="true">
      <rosparam>
-       cv_camera/device_id: 0
+       device_id: 0
```

Comment:
This can be one rosparam block (or better config file) and we could add a flag if we want to launch without the camera (e.g. if we use an rosbag). Similarly we should make the camera device id configurable via an launch argument.

Reply:
This is especially relevant for laptop with the USB webcam and an internal one. Editing the launch file for this is not so nice.